### DISCUSSION TITLE
Bug: GoPro Hero 9 Videos not detected correctly

### DISCUSSION THREAD

**Discussion by hatl** [photoprism/photoprism#1829] [Q&A]
**What does not work as expected?**

Video detected as JPG

**How can we reproduce it?**

The following file is part of the originals: GX010255.MP4
PhotoPrism adds .jpg to name: GX010255.MP4.jpg
Playback doesn't work.

Key | Value
-- | --
Storage Folder | Sidecar
Name | my/folder/GX010255.MP4.jpg
Size | 1920 × 1080, 56.3 KB
Camera | HERO9 Black

**What version you are using?**

211215-93b26f19-Linux-x86_64

**Comment by graciousgrey** [comment]
It's expected that a jpg is created for each video. This is the thumbnail you see in the views. 
What exactly is not working regarding the playback? Do you get errors or is it just slow?
In case playback is too slow, you might want to convert your videos upfront, see https://docs.photoprism.app/getting-started/docker-compose/#command-line-interface.

**Reply by hatl** [reply]
The player frame opens, but the video doesn't start.
There is no playback since PhotoPrism seems to try a "playback" of the jpg file instead of the mp4 file.
![Screenshot](https://user-images.githubusercontent.com/70505/147482989-80803d36-8410-43b3-81bb-918c9070e3f5.png)

Strange thing is, that it only happens to some files - others work fine.
Navigation to edit, then to the mp4 file and clicking download downloads the correct file.

**Reply by lastzero** [reply]
No, that's likely not the case. Are you using a proxy or specific browser that could cause the issues?
